src/openforms/formio/tests/validation/helpers.py

```diff
@@ -13,12 +14,12 @@
 
 
 def validate_formio_data(
-    component: Component, data: JSONValue
+    component: Component, data: JSONValue, extra_context: dict[str, Any] = {}
```

Comment:
can we not provide properly defined/typed kwargs for extra context? The structure of context is well-known since it's built when the validation is invoked.